website/docs/r/org_vdc_access_control.html.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: org_vdc_access_control"
+sidebar_current: "docs-vcd-resource-org-vdc-access-control"
+description: |-
+Provides a VMware Cloud Director Org Vdc access control resource. This can be
```

Comment:
check everywhere that `VDC` is not `Vdc`

Reply:
Yeah, checked and changed all visible to user vdc to VDC